portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,59 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    padding-left: 20px;
```

Comment:
I think these should be in alphabetical order (https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

Reply:
Got it, fixed in new commit.